### PR TITLE
HB-01

### PR DESCRIPTION
**Pull request by engineer027**
configured pom;

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,8 +9,78 @@
...
+        </maven.checkstyle.plugin.configLocation>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (oleksandrdziubiuk) [inline pom.xml]: Don’t add dependencies that you don’t use to pom.xml;
чеклист

**Review by oleksandrdziubiuk — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -9,8 +9,71 @@
...
+                        <source>${jdk.version}</source>
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
 </project>
```
Comment (Sofasmile) [inline pom.xml]: let's add an empty line at the end of the file

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    @Override
+    public Movie add(Movie movie) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            Long itemId = (Long) session.save(movie);
+            transaction.commit();
+            movie.setId(itemId);
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java]: https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_01.html

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert Movie entity ", e);
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java]:
```suggestion
            throw new DataProcessingException("Can't insert Movie entity " + movie, e);
```

File: src/main/resources/hibernate.cfg.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">1234</property>
+        <property name="show_sql">true</property>
+        <property name="hibernate.hbm2ddl.auto">create-drop</property>
+
+        <mapping class="com.dev.cinema.model.Movie"/>
+    </session-factory>
+
+</hibernate-configuration>
```
Comment (Sofasmile) [inline src/main/resources/hibernate.cfg.xml]: add empty line

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**